src/main/java/org/sopt/lequuServer/domain/favorite/model/Favorite.java

```diff
@@ -0,0 +1,36 @@
+package org.sopt.lequuServer.domain.favorite.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.sopt.lequuServer.domain.book.model.Book;
+import org.sopt.lequuServer.domain.member.model.Member;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "favorite")
+public class Favorite {
+
+    @Id
+    @Column(name = "favorite_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```
[P1] ManyToOne 관계에서는 기본적으로 지연로딩을 설정하는 것이 좋습니다. 안그러면 N+1 문제가 발생할 수 있어요!
이부분은 구글에 검색해보면 자세히 나올건데 간단히 설명드리면
1. ManyToOne 관계는 기본적으로 EAGER 즉, 즉시로딩이 기본 설정입니다.
2. 따라서 OneToMany 관계에 있는 데이터를 조회할 때 (예를 들면 Member), Member와 연결되어 있는 Favorite 객체도 같이 조회하려고 합니다.
3. 근데 Member 1개에는 여러개의 Favorite이 존재하죠?? 그러면 그 Favorite만큼 DB에 쿼리를 날려서 Favorite 데이터를 가져오게 됩니다.
4. 단지 Member를 1을 조회하려고 한 것 뿐인데 N번의 쿼리가 날라갔죠?? 이게 N+1 문제입니다.

그래서 지연로딩을 걸면 Member를 조회할 때 Favorite의 프록시 객체(이것도 궁금하면 찾아보세영 ㅋㅋ)가 주입되게 되고, 실제 Favorite을 조회하지 않는 이상 추가적인 쿼리가 발생하지 않게 됩니다!

이것도 따봉만 날리고 가봐라 현예진

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 왕 자세한 설명 감쟈합니다 ...!!!!!! 바로 반영할게유